### PR TITLE
Document to test releases with the Canton-based sandbox

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
This could not be achieved previously due to an issue that has
been fixed by https://github.com/DACH-NY/canton/pull/8475.

The next release should be tested again with the Canton-based
sandbox to ensure it works on both Windows and MacOS/Linux.

changelog_begin
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Thanks, we need a few more changes for things to work with the new gsg but I’ll take care of those in a separate PR.

**Comment by stefanobaghino-da** [comment]
> Thanks, we need a few more changes for things to work with the new gsg but I’ll take care of those in a separate PR.

I'll wait for this week's snapshot release to be tested (should be done today) before merging this PR, this would leave you one full week to add those changes without any likely misunderstanding on what to test. Thanks!

**Comment by cocreature** [comment]
> I'll wait for this week's snapshot release to be tested (should be done today) before merging this PR, this would leave you one full week to add those changes without any likely misunderstanding on what to test. Thanks!

no need to wait, we need those changes either way. The current instructions are just broken.

**Pull request merged**
